src/tabs/default.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  @include iui-tabs;
+
+  > li {
+    > button {
+      height: $iui-baseline * 3;
+      padding: 0 $iui-m;
+      box-sizing: border-box;
+      @include themed {
+        background-color: t(iui-color-background-2);
+        border: 1px solid t(iui-color-background-5);
+      }
+
+      .iui-label {
+        text-align: left;
+
+        .iui-description {
+          font-size: $iui-font-size-small;
+          @include themed {
+            opacity: t(iui-opacity-3);
+          }
+        }
+      }
+      
+
+      &.iui-active {
+        border-bottom-color: transparent;
+        @include themed {
+          box-shadow: inset 0 $iui-xxs 0 0;
+        }
+      }
+
+      &[disabled] {
+        @include themed {
+          background-color: t(iui-color-background-3);
+        }
+      }
+    }
+
+    &:not(:last-child) > button {
+      border-right: none;
+    }
+  }
+
+  &:not(.iui-borderless):not(.iui-pill) ~ .iui-tabs-content {
+    box-sizing: border-box;
+    padding-left: $iui-m;
+    padding-right: $iui-m;
+    margin-top: -1px;
```

Comment:
Why is -1 necessary?

Reply:
Without it we end up with this:
![Screen Shot 2021-06-15 at 4 40 10 PM](https://user-images.githubusercontent.com/849817/122120403-5cf3d800-cdf8-11eb-9305-eddb2fcbd751.png)

Reply:
Why not remove border-top from content then?

Reply:
Well then we end up with something like this.
![Screen Shot 2021-06-15 at 8 12 01 PM](https://user-images.githubusercontent.com/849817/122138999-f4b3ef00-ce15-11eb-85ca-3a9a75ee775c.png)